pallets/parachain-staking/src/round.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Constants and functions for computing round-related information
+use crate::{Config, Pallet};
+
+pub const SECONDS_PER_YEAR: u32 = 31557600;
+pub const SECONDS_PER_BLOCK: u32 = 12;
+pub const BLOCKS_PER_YEAR: u32 = SECONDS_PER_YEAR / SECONDS_PER_BLOCK;
+/// Number of blocks prior to next round when election results are computed and committed
+pub const EARLY_ELECTION_OFFSET: u32 = 1;
```

Comment:
```suggestion
pub const DEFAULT_EARLY_ELECTION_OFFSET: u32 = 1;
```
This is not mutable now, it is hardcoded. We could make this the default assignment but we could also add a `set_early_election_offset` dispatchable callable by root that could set the next early election offset such that it only takes effect next round.